### ISSUE TITLE
Mentor feedback taking too long

### ISSUE THREAD

**Issue by luizdepra** [exercism/exercism#4592]
Hey,

Before explaining my problem, thank you all for this awesome platform.

I'm training Rust through Exercism's Rust Track and now I'm stuck in the `Sublist` exercise because no mentor gave me feedback until now. I don't know if there is bug and my exercise fell in a limbo, or the track exercise queue is overloaded right now. I just want to understand what is happening and help if possible.

To help track any possible bug, these are the things I made so far (related to this exercise):

- I opened the `sublist` exercise and when away for while from Exercism (about a month, probably).
- I returned, and submitted my code (I thing it was in the end of November, 2018)
- This week I opened the exercise and saw that I should update it, and so I did.

That is it, and thank you again.

**Comment by luizdepra** [comment]
Yeah, all the tests passed. I forgot to mention that I tried to submit the solution again but no new iteration was added in the exercise.

I just updated the issue with my solution UUID and and these complementary information.

**Comment by luizdepra** [comment]
Still no response from a mentor. :cry:

**Comment by TheDarkula** [comment]
@luizdepra I'm on it :)
You can close this issue if you like.
